app/(content)/_layout.js

```diff
@@ -0,0 +1,86 @@
+import { Tabs } from "expo-router/tabs";
+import { useSegments } from "expo-router";
+
+export default function AppLayout() {
+  // comment out below for debug
+  // const segment = useSegments();
+  // console.log(segment); // although no longer using useSegments to define which tabbars are shown,
+  return (
+    <Tabs screenOptions={{ headerShown: false }}>
+      <Tabs.Screen
+        name="drill/index"
+        // show tab and tabbar
+        options={{
+          href: "/drill",
+          title: "Drills",
+          headerShown: false,
+        }}
+      />
+
+      <Tabs.Screen
+        name="drill/[id]/(segments)"
+        // hide tab, show tabbar
+        options={{
+          href: null,
+          title: "Drill ID index",
+          headerShown: false,
+        }}
+      />
+
+      <Tabs.Screen
+        name="drill/[id]/submission" // applies to all files in drill submission directory
+        // hide tab and tabbar
+        options={{
+          href: null,
+          title: "Drill submission directory",
+          headerShown: false,
+          tabBarStyle: {
+            display: "none",
+          },
+        }}
+      />
+
+      <Tabs.Screen

```

Comment:
I made an oversight of this in the figma. Can the user result have the tabbar?

Reply:
Did you mean `app\(content)\drill\[id]\users\[user].js`? I put tabbar back for that one. 

If you meant `app\(content)\drill\[id]\submission\result.js`, I can switch later